ddl/ddl_api.go

```diff
@@ -673,6 +673,17 @@ func getDefaultValue(ctx sessionctx.Context, col *table.Column, c *ast.ColumnOpt
 
 		return value, nil
 	}
+	// handle default next value of sequence. (keep the expr string)
+	str, isExpr, err := handleSequenceDefaultValue(c)
+	if isExpr {
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		col.DefaultIsExpr = true
```

Comment:
Names `isExpr` and `DefaultIsExpr` are not good. It may be some other expressions except for sequence expression, but it's still an expression.
You can rename it to `isSeqExpr` and `DefaultIsSeqExpr`.

Reply:
yeh, we don't set.
actually there is no other normal expression in default value.